### ISSUE TITLE
Just some ideas for more Mauls and Tridents (once we update to 1.13+)

### ISSUE THREAD

**Issue by Cruuk** [Tslat/Advent-Of-Ascension#128]
I know I don't usually make suggestions for new features, but I'm in the mood to just post random ideas for weapon types. I know it's a new weapon type but I'm hoping that Mauls get some more variety in the future once Scimi starts adding new stuff. I imagine it shouldn't be too difficult, look at archerguns, each of them have their own quirks but in the end they're all the same.

**Describe the new feature you think would be good to add**
__Mauls__
**[Lunar Maul]**
-Obtaining: with this crafting recipe.
![image](https://user-images.githubusercontent.com/38159897/48972217-ca418280-efda-11e8-837d-a96ac1ff27db.png)
-Regular damage for a Maul at this point
-Pretty High knockback, but it has a 20% chance to knock the enemy backwards instead.
-It looks like a maul but the blunt object is in the shape of a crescent moon that is pink.

**[Sledge]**
-Obtaining: with this crafting recipe.
![image](https://user-images.githubusercontent.com/38159897/48972277-e7c31c00-efdb-11e8-9cb0-78633dbb6afd.png)
(That's Blazium by the way)
-High damage for a maul at this point.
-High knockback.
-Sets the enemy on fire for a short time
-Gives slowness 1 to the user while being held (it's heavy!)
-Looks like a big sledgehammer with a red-ish tint to it.

**[Tremor]**
-Obtaining: dropped by Tyrosaur.
![image](https://user-images.githubusercontent.com/38159897/48972519-c6b0fa00-efe0-11e8-8586-b4d7eeded65b.png)
-Good damage for a maul at this point.
-Good knockback.
-10% chance to cripple all nearby enemies for a short time. (maul counterpart to Vulcammer)
-The blunt object looks like a gray/green rock with small DInosaur aesthetics on it.

**[Lollypop Maul]**
-Obtaining: with this crafting recipe.
![image](https://user-images.githubusercontent.com/38159897/48972366-9156dd00-efdd-11e8-9a7d-6ad6bf8ba652.png)
-Regular damage for a maul at this point.
-Pretty good knockback at this point.
-Right-clicking it allows you to eat it, it is comparable to cooked chicken in hunger/saturation, and has a slow eating speed. It is non-consumable, but doing this removes 10% of it's durability.
-It looks like a BIG LOLLYPOP.

(I decided to be sort of vague with the damage/kb because I think Scimi would do a better job than me of balancing this stuff if actually considers this)

__Tridents__
I know there's sort of been a debate on whether if AoA should expand on tridents (and I guess crossbows) or not, and I think the best course of action would be mainly to add a couple new Tridents once we get to 1.13, we already have plenty of weapon types so I think just a couple new additions for Tridents would be good.

**[Devil's pitchfork]**
-Obtained somewhere in the Nether.
-Cannot surf in water if enchanted with Riptide, instead it allows you to surf in lava (still need fire resist to utilize it without danger though), whether it should require the enchantment or not I don't know. Maybe it already has it at base but the enchantment just increases the velocity?

**[Lightning Rod]**
-Obtained somewhere in Iromine.
-Strikes lightning on hit.
I know vanilla tridents can already sort of do this with Chanelling while it's raining, but AoA already has swords that can set enemies on fire without fire aspect making this no different from that in context. This just removes the requirement for the rain.

**[Poseidon's Trident]**
-Obtained somewhere in L'Borean, or dropped by Pinchers instead?
-Basically just an upgrade to the vanilla trident.

**Include any additional background information**
Today I learned that 1.13 added an enchantment that's like Smite/Bane of Arthropods but for aquatic mobs, it's called "Impaling". You know what this means...
(It's also exclusive to tridents I think)

EDIT: Gave Lollypop Maul a much more interesting ability than just slowness.
